src/main/assembly/dist/cfg/application.properties

```diff
@@ -5,6 +5,9 @@ daemon.http.port=20160
 
 # terminating slash is required
 bag-store.url=http://localhost:20110/
+bag-store.connection-timeout-ms=600000
```

Comment:
nice! didn't know we hadn't any timeout specified on these yet 😉
Can you please also make these parameters configurable in `easy-dtap` (https://github.com/DANS-KNAW/easy-dtap/pull/430)?

Reply:
done